cmake/modules/FindUPNP.cmake

```diff
@@ -2,13 +2,15 @@
 # UPNP_FOUND - system has ZeroMQ lib
 # UPNP_INCLUDE_DIR - the ZeroMQ include directory
 # UPNP_LIBRARY - Libraries needed to use ZeroMQ
+include(BrewHelper)
+find_brew_prefix(BREW_HINT_UPNP miniupnpc)
```

Comment:
Write homebrew-related code in a code block that has been confirmed to be for macOS like bellow:

https://github.com/bitcoin/bitcoin/blob/master/cmake/module/FindBerkeleyDB.cmake#L31-L33

Please modify FindBerkeleyDB.cmake in the same way.